workflows/region.go

```diff
@@ -0,0 +1,41 @@
+package workflows
+
+import (
+	"github.com/temporalio/cloud-samples-go/protogen/temporal/api/cloud/cloudservice/v1"
+	"github.com/temporalio/cloud-samples-go/workflows/activities"
+	"go.temporal.io/sdk/worker"
+	"go.temporal.io/sdk/workflow"
+)
+
+const (
+	// region workflow types
+	GetRegionWorkflowType     = workflowPrefix + "get-region"
+	GetAllRegionsWorkflowType = workflowPrefix + "get-regions"
```

Comment:
[Nit] GetRegions, or rename suffix to say "get-all-regions"

Reply:
fixed, thanks